Rendering/Core/vtkLabeledContourMapper.cxx

```diff
@@ -603,6 +603,7 @@ bool vtkLabeledContourMapper::PrepareRender(vtkRenderer *ren, vtkActor *act)
       continue;
       }
     metric.Value = scalars->GetComponent(ids[0], 0);
+    metric.Value = std::fabs(metric.Value) > 1e-6 ? metric.Value : 0.0;
```

Comment:
@sankhesh should this be `std::fabs(metrics.Value) < 1e-6 ? metrics.Value : 0.0;` instead - should the greater than be a less than?

So if less than 1e-6, reset value to 0.0?